layers/+lang/lean/packages.el

```diff
@@ -0,0 +1,39 @@
+;;; packages.el --- Lean Layer packages File for Spacemacs
+;;
+;; Copyright (c) 2012-2014 Sylvain Benner
+;; Copyright (c) 2014-2015 Sylvain Benner & Contributors
+;;
+;; Author: Sylvain Benner <sylvain.benner@gmail.com>
+;; URL: https://github.com/syl20bnr/spacemacs
+;;
+;; This file is not part of GNU Emacs.
+;;
+;;; License: GPLv3
+(setq lean-packages '(mmm-mode f dash-functional
+                      (lean-mode :location (recipe
+                                            :fetcher github
+                                            :repo leanprover/lean
+                                            :files ("src/emacs/*.el")))))
+(defun lean/init-f ())
+
+(defun lean/init-dash-functional ())
+
+(defun lean/init-mmm-mode ())
+
+(defun lean/init-lean-mode ()
+  (use-package lean-mode
+    :defer t
+    :config
+    (evil-leader/set-key-for-mode 'lean-mode
```

Comment:
Would be better to replace with new macro `spacemacs/set-leader-keys-for-major-mode` instead of `evil-leader/set-key-for-mode` just to make new code consistent.